src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
```

Comment:
Consider adding a check to ensure that the `document` object is available before attempting to create elements. This can prevent errors in environments where the DOM might not be fully loaded.